### PR TITLE
Gitpod Jetbrains Launch

### PR DESCRIPTION
**Pull request by JohannesLandgraf**
## Related Issue(s)

Fixes #

## Release Notes

```release-note
```

## Documentation

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1963"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1963"></a>

### CODE REVIEW

File: src/routes/blog/gitpod-jetbrains.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+With JetBrains' [remote development announcement](https://www.jetbrains.com/remote-development/), we now seamlessly integrate and bundle with their standalone Gateway application. This means we'll run an instance of your favourite JetBrains IDE on your Gitpod ephemeral developer environment. All indexing, compiling and language processing happens in the cloud, while a thin client runs locally and provides the rich user experience you're already familiar with from your desktop IDE. To enable this experience we have internally built a [SSH gateway](https://www.gitpod.io/blog/copy-paste-ssh-workspace-accessa) (which also makes our vim and emacs users happy), re-designing our loading screen and expanding our /preferences to enable both Desktop and Browser based workflows. The result is available in today's beta release.
```
Comment (akosyakov) [inline src/routes/blog/gitpod-jetbrains.md:41]: a link to SSH Gateway is broken, should be https://www.gitpod.io/blog/copy-paste-ssh-workspace-access

File: src/routes/blog/gitpod-jetbrains.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+![02](../../../static/images/blog/gitpod-jetbrains/02.jpg)
+
+## Getting started in 5 minutes
+
+If you are interested in setting up a new Spring Boot project using Gitpod and JetBrains Gateway we wrote a detailed guide together with JetBrains. If you want to start your own project with Gitpod in your JetBrains IDE read on or refer to our [documentation](https://www.gitpod.io/docs/ides-and-editors/jetbrains-gateway).
+
+`youtube: 8djaRYT2FAY`
+
+1. **Install [JetBrains Gateway](https://www.jetbrains.com/help/idea/remote-development-a.html#gateway)**
+2. **Install the [Gitpod plugin](https://plugins.jetbrains.com/plugin/18438-gitpod-gateway)** - Navigate to the JetBrains Gateway preferences page and select the plugins tab to search and install the Gitpod plugin.
```
Comment (akosyakov) [inline src/routes/blog/gitpod-jetbrains.md:74]: I think it is outdated after announcement in the blog post as well in our docs. cc @loujaybee 

There will be install Gitpod on the welcome screen of Gateway. One does not need to go to preferences.

File: src/routes/blog/gitpod-jetbrains.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+3. **Update your [Gitpod preferences](https://gitpod.io/preferences)** - Select your preferred JetBrains IDE on the Gitpod preferences page which will set your default Desktop IDE for future workspace starts.
```
Comment (akosyakov) [inline src/routes/blog/gitpod-jetbrains.md:75]: there is no anymore `default Dekstop IDE`.

**Review by Cahllagerfeld — COMMENTED** [review]
We probably need to merge this branch once it's approved into #1421, because the new blogpost is referenced in that PR, and causes the build to fail because it's not present in that PR.

**Comment by JohannesLandgraf** [comment]
I added some fixes around links/language. The only thing remaining from my end is to:

- [ ]  Update some of the product screenshots @chrifro   
- [ ] Add link to blog post under Getting Started section @Nancy-Chauhan 
- [ ] Avoid merge conflicts with https://github.com/gitpod-io/website/pull/1421 @Cahllagerfeld

File: src/routes/blog/gitpod-jetbrains.md (hunk 2 above)
Comment (loujaybee) [reply src/routes/blog/gitpod-jetbrains.md]: Updated in: 98652908fb02d77b3b27c06b2faf21996ab363f9

File: src/routes/blog/gitpod-jetbrains.md (hunk 3 above)
Comment (loujaybee) [reply src/routes/blog/gitpod-jetbrains.md]: Updated in: 98652908fb02d77b3b27c06b2faf21996ab363f9

**Review by loujaybee — APPROVED** [review]

**Comment by loujaybee** [comment]
Looking to merge this now, any objections @JohannesLandgraf, @Cahllagerfeld, @phimae, @akosyakov ?

**Pull request merged**